CHANGELOG.md

```diff
@@ -8,6 +8,20 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 For compatibility information see `govukFrontendVersion` and `hmrcFrontendVersion` in
 [LibDependencies](project/LibDependencies.scala)
 
+## [1.20.0] - 2021-10-13
+
+### Added
+
+- Added `withHeading` and `withHeadingAndSectionCaption` helper methods for form inputs
+- Added various implicit conversions for `String` to nested case classes via `RichStringSupport`
+- Added various implicit conversions for `Seq[(String, String)]` to nested case classes via `RichSeqStringTupleSupport`
+- Added `HmrcYesNoRadioItems` helper
```

Comment:
@JoPintoPaul it all looks good to me, should put anything extra in the README about the tuple implicits and radio items helper? I'm not sure, since they're not complicated and the tests for them are clear so it's easy to find examples. The only bit is that I merged my dependency update in so this needs to be 1.21.0 now, apologies!